datasets/no2.data.mdx

```diff
@@ -79,116 +78,53 @@ layers:
+    ## Data Summary
 
-For nitrogen dioxide levels related to COVID-19, NASA uses data collected by the joint NASA-Royal Netherlands Meteorological Institute (KNMI) [Ozone Monitoring Instrument (OMI)](https://aura.gsfc.nasa.gov/omi.html) aboard the Aura satellite, as well as data collected by the Tropospheric Monitoring Instrument (TROPOMI) aboard the European Commission’s Copernicus Sentinel-5P satellite, built by the European Space Agency.
+    - **Temporal Extent:** January 2016 - Ongoing
+    - **Temporal Resolution:** Monthly
+    - **Spatial Extent:** Global
+    - **Spatial Resolution:** 0.1 degrees x 0.1 degrees
+    - **Data Units:** Molecules of nitrogen dioxide per square centimeter (Molecules NO<sub>2</sub>/cm2)
+    - **Data Type:** Research
 
-OMI, which launched in 2004, preceded TROPOMI, which launched in 2017. While TROPOMI provides higher resolution information, the longer OMI data record provides context for the TROPOMI observations.
+    </Prose>
 
-Scientists will use these data to investigate how travel bans and lockdown orders related to the novel coronavirus are impacting regional air quality and chemistry, as well as why these restrictions may be having inconsistent effects on air quality around the world.
-</Prose>
 </Block>
-
 <Block>
-<Figure>
-  <Image
-  src={new URL('./no2-south-america.jpg', import.meta.url).href}
-  alt='NO2 levels over South America from the Ozone Monitoring Instrument'
-  />
-  <Caption
-    attrAuthor='NASA'
-    attrUrl='https://nasa.gov/'
-  >
-    NO2 levels over South America from the Ozone Monitoring Instrument. The dark green areas in the northwest indicate areas of no data, most likely associated with cloud cover or snow.
-  </Caption>
-</Figure>
-<Prose>
-## Interpreting the data
-Nitrogen dioxide has a relatively short lifespan in the atmosphere. Once it is emitted, it lasts only a few hours before it dissipates, so it does not travel far from its source.
+  <Prose>
+    ## Source Data Access
+    The monthly tropospheric NO<sub>2</sub> data can be accessed here:
+    [https://avdc.gsfc.nasa.gov/pub/data/satellite/Aura/OMI/V03/L3/OMNO2d_HR/OMNO2d_HRM/](https://avdc.gsfc.nasa.gov/pub/data/satellite/Aura/OMI/V03/L3/OMNO2d_HR/OMNO2d_HRM/)
 
-Because nitrogen dioxide is primarily emitted from burning fossil fuels, changes in its atmospheric concentration can be related to changes in human activity, if the data are properly processed and interpreted.
+          ## Acknowledgment
+          Acknowledgement of the teams/organizations that contributed to the product. This can often be found in dataset documentation from source
```

Comment:
👀

Reply:
I don't know what to add here. It is written in the document. I commented in the google doc, who is suppose to write it.